desktop/src/roles/role-preview/role-preview-dialog.component.html

```
@@ -0,0 +1,55 @@
+                @if (row.description) {
+                  <span class="row-desc">{{ row.description }}</span>
+                }
+              </div>
+            }
+          </div>
+        </div>
+      }
+    }
+
+    <div class="actions">
+      <button mat-stroked-button mat-dialog-close>Close</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)